cms/models_test.py

```diff
@@ -372,6 +372,18 @@ def test_certificate_index_page(rf):
             request, "00000000-0000-0000-0000-000000000000"
         )
 
+    # Testing invalid UUIDs
+    with pytest.raises(Http404):
+        certifcate_index_page.bootcamp_certificate(request, "00000000-0000-")
+
+    with pytest.raises(Http404):
+        certifcate_index_page.bootcamp_certificate(request, "")
+
+    with pytest.raises(Http404):
+        certifcate_index_page.bootcamp_certificate(
+            request, "00000000-0000-0000-0000-000000000000000"
+        )
+
```

Comment:
The test can run on broader data set. Also please use parametrized tests in cases where the code is redundant. You can take a look at https://github.com/mitodl/mitxpro/pull/2990 for reference.